crates/fuel-core/src/executor.rs

```diff
@@ -779,7 +779,7 @@ where
                 *mint.mint_amount(),
             )
             .map_err(|e| anyhow::anyhow!(e))
-            .map_err(ExecutorError::CoinbaseCantIncreaseBalance)?;
+            .map_err(ExecutorError::CoinbaseCannotIncreaseBalance)?;
```

Comment:
It looks like you're replacing `Coinbase` with `Mint` - should this be `MintCannotIncreaseBalance`?

Reply:
Nope, because here the functionality of the coinable works incorrectly, instead of the problem with `Mint` transaction validity=)